### ISSUE TITLE
Ping Discord users when event positions are posted.

### ISSUE THREAD

**Issue by MikeRomaa** [Houston-ARTCC/zhu-frontend#3]
It would be nice to have the webhook mention all controllers on an event when the positions are posted to `event-rosters`.

## Depends On

- [ ] Waiting on VATUSA to add Discord IDs to the `/user/<cid>/` endpoints.
